### DISCUSSION TITLE
Unable to re-verify student membership

### DISCUSSION THREAD

**Discussion by ericmconnelly** [community/community#131370] [GitHub Education]
### Hi everyone,

My Github Student Membership is about to expire. I have tried many many times to re-verify my Github Student Membership but the system kept telling my that my application is denied even though I have uploaded valid documents. Below is the message

> Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on Jul 7, 2024 for [@ericmconnelly](https://github.com/ericmconnelly).

**Comment by mustafaobaidd** [comment]
1. **Ensure All Requirements Are Met**:
   - Verify that you are enrolled in a degree or diploma-granting course of study.
   - Make sure your documentation is up-to-date and clearly shows your name, school name, and the current date.

2. **Check Your Documents**:
   - Double-check that the documents you upload are in an acceptable format (PDF, JPG, PNG).
   - Ensure that your documents are clear, legible, and contain all the necessary information.

3. **Contact GitHub Support**:
   - If you have verified that all the requirements are met and the issue persists, contact GitHub Education Support directly.
   - Explain your situation clearly and provide all the relevant details and documents.

4. **Alternative Documents**:
   - If possible, try using different types of documents such as a recent transcript, a tuition bill, or an official letter from your institution confirming your enrollment.

5. **Retry with Different Account Details**:
   - If the issue is related to your account details, consider updating your account information or even creating a new GitHub account and trying the verification process again.

6. **Reach Out to Your Institution**:
   - Sometimes the problem might be with how your institution is recognized by GitHub. Contact your school's administration to ensure they are listed properly and see if they can assist you in verifying your student status.

**Reply by ericmconnelly** [reply]
I have tried all of these. None of them is helpful

**Reply by mustafaobaidd** [reply]
send to GitHub Support, this the draft of an email:
Subject: Issue with Re-verification of GitHub Student Membership

Body:

Dear GitHub Support Team,

I hope this message finds you well. I am writing to seek assistance with the re-verification of my GitHub Student Membership. Despite multiple attempts to upload valid documents, my application continues to be denied. Below are the details of my situation:

GitHub Username: [Your GitHub Username]
Email Address: [Your Email Address]
Institution Name: [Your Institution Name]
I have ensured that the documents I am submitting are clear, legible, and contain all the necessary information such as my name, institution name, and date. I have attached the documents I have been trying to submit, along with screenshots of the error messages I received during the application process.

I would greatly appreciate your assistance in resolving this issue as my current membership is about to expire. Thank you for your time and support.

Best regards,
[Your Full Name]

**Reply by ericmconnelly** [reply]
Ok I will try go through https://support.github.com/
